engine/src/cmdsf.cpp

```diff
@@ -2734,7 +2820,11 @@ void MCImport::compile(MCSyntaxFactoryRef ctxt)
 			MCSyntaxFactoryExecMethod(ctxt, kMCInterfaceExecImportSnapshotOfScreenMethodInfo);
 		}
 	}
-	else
+	else if (format == EX_OBJECT)
+    {
+        // COMPILE-TODO
```

Comment:
That code isn't used at the moment - it is part of the 'compile' stuff for later use when translating scripts.